.github/workflows/Build.yaml

```diff
@@ -19,17 +19,17 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
By any chance, can we just revert this back to 20.04?

I'm not sure if Runner upgrades are persistent, but, if they're not, then we might be using up a lot more CI minutes.

Reply:
That's possibly what I will try next.